From 3f1d5e778936e17fd4bed98b37f0c62d3a70e253 Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Sat, 6 Oct 2018 10:07:54 +0530
Subject: [PATCH 0442/1851] drm: xlnx: zynqmp_disp: Fixing potential
 segmentation fault

commit 42d258ae037b4a73917f8307a5ece13915c4952d from
https://github.com/Xilinx/linux-xlnx.git

If info is NULL, its member shouldn't be accessed.

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/zynqmp_disp.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/zynqmp_disp.c b/drivers/gpu/drm/xlnx/zynqmp_disp.c
index 14b894769e4b..81b9b2513536 100644
--- a/drivers/gpu/drm/xlnx/zynqmp_disp.c
+++ b/drivers/gpu/drm/xlnx/zynqmp_disp.c
@@ -2483,8 +2483,7 @@ static int zynqmp_disp_plane_mode_set(struct drm_plane *plane,
 	int ret;
 
 	if (!info) {
-		dev_err(dev, "unsupported framebuffer format %s\n",
-			drm_get_format_name(info->format, &format_name));
+		dev_err(dev, "No format info found\n");
 		return -EINVAL;
 	}
 
-- 
2.31.1

